components/cell/jwks-server/main.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package main
+
+import (
+	"github.com/cellery-io/mesh-security/components/cell/jwks-server/crypto/service"
+	"log"
+)
+
+func main() {
+	go service.UnSecuredService()
+	err := service.SSLSecuredService()
+	if err != nil {
+		log.Printf("Error occure while establishing the SLL service. %s", err)
```

Comment:
SLL, occure ? Also better it it would have been "Error occurred while establishing connection with SSL service"

Reply:
Fixed by #75